### PR TITLE
Handle unary minus and plus in lists during convert.

### PR DESCRIPTION
**Pull request by chriseppstein**
Fixes Issue #822

### CODE REVIEW

File: lib/sass/script/list.rb [hunk 1]
```diff
@@ -49,7 +49,8 @@ def to_sass(opts = {})
       return "()" if value.empty?
       precedence = Sass::Script::Parser.precedence_of(separator)
       value.reject {|e| e.is_a?(Null)}.map do |v|
-        if v.is_a?(List) && Sass::Script::Parser.precedence_of(v.separator) <= precedence
+        if v.is_a?(List) && Sass::Script::Parser.precedence_of(v.separator) <= precedence ||
+          separator == :space && v.is_a?(UnaryOperation) && (v.operator == :minus || v.operator == :plus)
```
Comment (nex3) [inline lib/sass/script/list.rb]: Style nit: Indent this line two more spaces.

File: lib/sass/script/unary_operation.rb [hunk 2]
```diff
@@ -4,6 +4,10 @@ module Sass::Script
   #
   # Currently only `-`, `/`, and `not` are unary operators.
   class UnaryOperation < Node
+
```
Comment (nex3) [inline lib/sass/script/unary_operation.rb]: Style nit: remove this newline.

File: lib/sass/script/unary_operation.rb [hunk 3]
```diff
@@ -4,6 +4,10 @@ module Sass::Script
...
+    attr_reader :operator
+    attr_reader :operand
```
Comment (nex3) [inline lib/sass/script/unary_operation.rb:11]: Move the documentation that's currently attached to the parameters to these attributes, and replace the param documentation with links to the attributes (as in e.g. funcall.rb).

**Comment by nex3** [comment]
A few comments, otherwise :thumbsup:

File: lib/sass/script/list.rb (hunk 1 above)
Comment (chriseppstein) [inline lib/sass/script/list.rb]: fixed.

File: lib/sass/script/unary_operation.rb (hunk 2 above)
Comment (chriseppstein) [inline lib/sass/script/unary_operation.rb]: fixed.

File: lib/sass/script/unary_operation.rb (hunk 3 above)
Comment (chriseppstein) [inline lib/sass/script/unary_operation.rb:11]: done.

**Comment by nex3** [comment]
22370c84a7b6617813436fa3fd952f89a7821528 reverts this to get the tests green. Feel free to re-merge when the tests are fixed.

**Comment by chriseppstein** [comment]
regression bug on master is fixed closed by 7ff8fef

**Pull request closed without merging**